ldi-orchestrator/ldio-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/pipeline/creation/LdioObserver.java

```diff
@@ -69,6 +70,13 @@ public void increment() {
 		Metrics.counter(LDIO_DATA_IN, PIPELINE_NAME, pipelineName, LDIO_COMPONENT_NAME, componentName).increment();
 	}
 
+	public boolean hasProcessedAllData() {
+		final double dataIn = Metrics.counter(LDIO_DATA_IN, PIPELINE_NAME, pipelineName, LDIO_COMPONENT_NAME, componentName).count();
+		final double dataOut = Metrics.counter(LDIO_DATA_OUT, PIPELINE_NAME, pipelineName).count();
+		log.atInfo().log("Received data: {} - Sent data: {}", dataIn, dataOut);
```

Comment:
now this i really don't like :p this will flood the logs like hell

Reply:
Ah oops, was something during development and I forgot to delete it or set it at debug level 🙈